integrationtest/src/main/scala/io/gearpump/integrationtest/minicluster/RestClient.scala

```diff
@@ -193,7 +193,8 @@ class RestClient(host: String, port: Int) {
 
   def restartApp(appId: Int): Boolean = try {
     val resp = callApi(s"appmaster/$appId/restart", CRUD_POST)
-    upickle.default.read[Status](resp).success
+    upickle.default.read[Status](resp)
+    true
```

Comment:
As the actor system returns `{success: false, reason: "timeout..."}`, please open an issue to confirm, whether this is a reasonable response.

And leave a comment with the issue number in code, so that code reader will understand, why we do not return `status.success` but true. And we might fix it later.